### DISCUSSION TITLE
Feature discussion: handling compressed .apk's (.zip)

### DISCUSSION THREAD

**Discussion by kenderipa** [ImranR98/Obtainium#884] [Ideas]
Hello again,

some of the github repositories publish their releases as .zip's. Would it make sense for Obtainium to handle them?

Example: https://github.com/XProger/OpenLara/releases/tag/latest

Thanks!

**Comment by kenderipa** [comment]
Also, with regard to https://github.com/ImranR98/Obtainium/issues/295, perhaps only the links containing "android" should be requested for unzipping?

**Comment by kenderipa** [comment]
Any feedback?

**Comment by kenderipa** [comment]
Well, I've looked at  https://github.com/marcosnils/bin, and it has a pretty neat feature: --all flag, that lists all the files a linux binary could be installed from and lets the user decide, which file to pick. Could it be implemented in Obtainium in scope of this discussion? Like list all the files, pick an Android .zip, unzip it and install?

**Comment by ImranR98** [comment]
There's already an issue open for this (#983) so any additional discussion should go there. Regarding user choice, it should be possible to let the user specify a regex filter for zip files, the same way we already do for APKs (the source code zip would be ignored automatically).
